packages/attest/src/store.ts

```diff
@@ -29,7 +29,11 @@ export const writeAttestation = async (
       owner: github.context.repo.owner,
       repo: github.context.repo.repo,
       headers: options.headers,
-      data: {bundle: attestation}
+      bundle: attestation as {
```

Comment:
Is this just to satisfy compile time type assertions?

Reply:
yup exactly